editoast/src/views/pagination.rs

```diff
@@ -9,22 +10,28 @@ use diesel_async::{AsyncPgConnection as PgConnection, RunQueryDsl};
 use serde::Deserialize;
 use serde::Serialize;
 use thiserror::Error;
+use utoipa::{IntoParams, ToSchema};
 
 use editoast_derive::EditoastError;
 
 /// A paginated response
-#[derive(Debug, Serialize)]
+#[derive(Debug, Serialize, ToSchema)]
+#[aliases(PaginatedInfras = PaginatedResponse<InfraWithState>)]
```

Comment:
Note to reviewer: required to inform utoipa which types we intend to specialize this schema with.